draft-ietf-acme-acme.md

```diff
@@ -1178,16 +1180,17 @@ Content-Language: en
 
 ### External Account Binding
 
-The server MAY require a value to be present for the "external-account-binding"
-field.  This can be used to an ACME account with an existing account in a
-non-ACME system, such as a CA customer database.
+The server MAY require a value for the "external-account-binding" field to be
+present in "new-account" requests.  This can be used to an ACME account with an
```

Comment:
Typo: "This can be used to an ACME account" - missing a verb here, probably "bind"? e.g. "This can be used to bind an ACME account with ..."

Reply:
This should be fixed in #357